packages/tests/index.js

```diff
@@ -57,3 +64,21 @@ async function getAdder(adderName) {
 
     return adder;
 }
+
+const cwd = path.resolve(fileURLToPath(import.meta.url), "..");
+
+// We're using `execSync` instead of our `executeCli` because we need the cleanup to be synchronous
+function startDocker() {
+    console.log("Starting docker containers");
+    execSync("docker compose up --detach", { cwd, stdio: "inherit" });
```

Comment:
Anything that would prevent us from using `stdio: "pipe"`.

As of now the test outputs, especially in the GitHub pipeline, are completely cluttered with layer download information.